lib/SPIRV/LLVMToSPIRVDbgTran.cpp

```diff
@@ -1196,6 +1216,10 @@ SPIRVEntry *LLVMToSPIRVDbgTran::transDbgFunction(const DISubprogram *Func) {
     DebugFunc = transDbgTemplateParams(TPA, DebugFunc);
   }
 
+  if (isNonSemanticDebugInfo() &&
+      (Func->isMainSubprogram() || IsEntryPointKernel))
+    transDbgEntryPoint(Func, DebugFunc);
```

Comment:
I just realized, that for this case and for the case below - transDbgFuncDefinition, static analyzer tools will complain, that result of the functions is unused. Can we make these two functions returning void? (can be done it this PR)

Reply:
It's possible to make these two functions return void. However they call `BM->addDebugInfo` inside, which does return the value - so the problem for static analysis will still exist. Not sure how to resolve this properly

Reply:
okay, lets do not change result type to void. Instead we can add
 [[maybe_unused]] auto *Inst = transDbgEntryPoint